src/schema/rules/sidecars.yml

```
@@ -61,21 +63,21 @@ MRISequenceSpecifics:
 
 PETMRISequenceSpecifics:
     selectors:
-        - modality == "MRI"
-        - "PET" in dataset.modalities
+    - modality == "MRI"
+    - PET in dataset.modalities
```

What about the inverse?

```suggestion
    - dataset.modalities contains "PET"
```

---

That works for me.